### ISSUE TITLE
deprecation warning

### ISSUE THREAD

**Issue by hboetes** [voxpupuli/puppet-zypprepo#21]
When using your module I get this warning with puppet-agent-1.10.4-1.sles11
```
Warning: Creating zypprepo via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead.
   (at /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet.rb:227:in `newtype')
```

**Comment by alexjfisher** [comment]
@hboetes Unless I'm missing something, this was fixed in https://github.com/voxpupuli/puppet-zypprepo/pull/13 which was included in the 2.0.0 release.  As such, I will close this issue. (Please reopen if you think the issue wasn't correctly fixed).

The module was fairly recently migrated to Vox Pupuli.  Perhaps you're using [darin/zypprepo](https://forge.puppet.com/darin/zypprepo) which stops at release 1.0.2?  If so, consider migrating to Vox Pupuli's [puppet/zypprepo](https://forge.puppet.com/puppet/zypprepo) to get the 2.0.0 release.

**Comment by hboetes** [comment]
Thanks for the heads up, Alex!
